parts/kuberneteswindowssetup.ps1

```diff
@@ -177,7 +177,7 @@ Write-KubernetesStartFiles($podCIDR)
 {
     $KubeletArgList = @("--hostname-override=`$global:AzureHostname","--pod-infra-container-image=kubletwin/pause","--resolv-conf=""""""""","--api-servers=https://`${global:MasterIP}:443","--kubeconfig=c:\k\config")
     $KubeletCommandLine = @"
-c:\k\kubelet.exe --hostname-override=`$global:AzureHostname --pod-infra-container-image=kubletwin/pause --resolv-conf="" --allow-privileged=true --enable-debugging-handlers --api-servers=https://`${global:MasterIP}:443 --cluster-dns=`$global:KubeDnsServiceIp --cluster-domain=cluster.local  --kubeconfig=c:\k\config --hairpin-mode=promiscuous-bridge --v=2 --azure-container-registry-config=c:\k\azure.json --runtime-request-timeout=10m
+c:\k\kubelet.exe --hostname-override=`$global:AzureHostname --pod-infra-container-image=kubletwin/pause --resolv-conf="" --allow-privileged=true --enable-debugging-handlers --api-servers=https://`${global:MasterIP}:443 --cluster-dns=`$global:KubeDnsServiceIp --cluster-domain=cluster.local  --kubeconfig=c:\k\config --hairpin-mode=promiscuous-bridge --v=2 --azure-container-registry-config=c:\k\azure.json --runtime-request-timeout=10m  --cloud-provider=azure --cloud-config=c:\k\azure.json

```

Comment:
Does this switch work for all versions from 1.5 - 1.7?

Reply:
@JiangtianLi it works for all versions, before v1.8, cloud-provider & cloud-config is not used in k8s kubelet in windows, I also did the test, it works well.

Reply:
@andyzhangx So mean --cloud-provider and --cloud-config exist all for version 1.5 and up but are no-op before 1.8? I'll sign-off then. Thanks.

Reply:
> So mean --cloud-provider and --cloud-config exist all for version 1.5 and up but are no-op before 1.8? 

Yep!

Reply:
@JiangtianLi yes, and it's not before **v1.8.0**, I have another PR(azure disk on windows, https://github.com/kubernetes/kubernetes/pull/51252) which depends on this config,  that's why I created this acs-engine config change PR. And https://github.com/kubernetes/kubernetes/pull/51252 would be likely into v1.8.1, so let's say k8s v1.8.1 depends on this cloud config.